src/main/cli/configs.py

```diff
@@ -3,6 +3,13 @@
 from enum import Enum
 from typing import List
 
+from src.main.preprocessing.inefficient_statements_removing import remove_inefficient_statements
+from src.main.preprocessing.int_experience_adding import add_int_experience
+from src.main.preprocessing.intermediate_diffs_removing import remove_intermediate_diffs
+from src.main.preprocessing.preprocessing import preprocess_data
+from src.main.splitting.splitting import split_tasks_into_separate_files
+from src.main.splitting.tasks_tests_handler import run_tests
```

Comment:
Imports order (we definitely need an automatic tool for that or reconsidering our imports style)